static/javascript/tba_js/tba.js

```diff
@@ -153,3 +153,13 @@ $(document).ready(function(){
     $("#robot-image").html('<div class="thumbnail carousel team-media-carousel"><a href="' + url + '" target="_blank"><img src="' + url + '" alt="April Fools!"></a></div>');
   }
 });
+
+// Init Firebase
+var config = {
```

Comment:
We shouldn't check in the api keys, firebase domain, and storage bucket. Can we have some placeholders here, and then do some regex in a paver task that replaces them with the real values at deploy time?